tox.ini

```
@@ -2,8 +2,8 @@
 envlist =
     py{38,39,310}-django32
     py{38,39}-django42
-    py{310,311}-django{42,50,main}
-    py312-django{42,50,main}
+    py{310,311}-django{42,50,51,main}
+    py312-django{42,50,51,main}
```

I think we can simplify this to

```suggestion
    py{310,311,312}-django{42,50,51,main}
```

---

good point, done!